easybuild/easyblocks/generic/conda.py

```diff
@@ -64,17 +64,25 @@ def install_step(self):
         if (get_software_root('anaconda2') or get_software_root('miniconda2') or
                 get_software_root('anaconda3') or get_software_root('miniconda3')):
             conda_cmd = 'conda'
-        elif get_software_root('mamba'):
-            conda_cmd = 'mamba'
         elif get_software_root('micromamba'):
             conda_cmd = 'micromamba'
+        elif get_software_root('mamba'):
+            conda_cmd = 'mamba'
         else:
-            raise EasyBuildError("No conda/mamba/micromamba available.")
+            raise EasyBuildError("No conda/micromamba/mamba available.")
 
-        # initialize conda environment
+        # initialize conda environment, except for micromamba
         # setuptools is just a choice, but *something* needs to be there
-        cmd = "%s config --add create_default_packages setuptools" % conda_cmd
-        run_cmd(cmd, log_all=True, simple=True)
+        force = '--force'
+        if conda_cmd == 'micromamba':
+            force = ''
+            try:
+                os.rmdir(self.installdir)
```

Comment:
better use `easybuild.tools.filetools.remove_dir`, which already takes care of the try-except for you.